### ISSUE TITLE
3.1.0 Finalization: TODO/Information

### ISSUE THREAD

**Issue by artdude543** [DarkPacks/SevTech-Ages#3234]
These are the list of issues/changes to be made to the pack before 3.1.0 is ready to go.
_TLDR: The pre-flight checklist for the update_

## Config/Script
- [x] Compress Cobblestone needs checking over as some recipes derped out during refactor.
- [x] Meka Crusher needs recipe checks as some are missing which are on the manual ones which are not ideal for later ages Fixed: https://github.com/DarkPacks/SevTech-Ages/commit/cb0b4dae01feba1d3f3480d278efa0caa1e16143
- [x] `[triumph]: Could not build advancement triumph:stage3/viescraft: vc:airship_ignition is not a valid item resource location.` Fixed: https://github.com/DarkPacks/SevTech-Ages/commit/95313ce519152033d53ec1b6ee7a9d9a30662f04
- [x] `[Galacticraft]: [config] External Sealable IDs: unrecognised block name 'nex:block_glass_soul:0'.` Fixed: https://github.com/DarkPacks/SevTech-Ages/commit/2b7be228a8e08660a4dae8dc3aa0dad6656f337f

All the issues logged under the [milestone](https://github.com/DarkPacks/SevTech-Ages/issues?q=is%3Aopen+is%3Aissue+milestone%3A3.1.0) for 3.1.0 need to be looked over. To either be fixed/resolved or tagged to the next update.

Issue #2989 needs to be reviewed to see if any the suggestions can be applied to this update. Or will be changed in the "DLC" update later this year.

## Details
We will also have a few beta test rounds to ensure nothing is broken/missed during the refactor. As any-thing could of been changed. So in regards to that anyone who is on our Discord and are marked as a `tester` you'll have access to the next update soon, along with access to a development server.

We are not going to accept any new testers for this round. But on the next update we'll be looking to expand the pool to help test the next major update or "DLC" as it's going to be tagged as.

So testers please refer to the secret channel on our Discord for the details on the pack update. Along with reporting of issues to here or there (as long as you cover the details we need)

This update has been months in the work, it has so many fixes internally along with mod fixes. From early testing performance has improved a lot! So for the potatoes out there, you'll have some more chips per frame! 😄

**Comment by TheSnowyChickens** [comment]
Hi,
I have a question, maybe its not the best place for this, but I have to say, that we rly need the new version.
On a multiplayer Server we have BIG problems with crashes, time outs (dimension change / log in) and many more.
its no fun to play, if we can't play for a long time without negative background.

**Comment by artdude543** [comment]
You just need to wait for the update like all the other players of the pack. There is no eta on the update and their  won't be.

**Comment by Pandoxy** [comment]
its been at 87% for almost a month xd

**Comment by artdude543** [comment]
So? Rome was not built in a day. So the pack will be updated when we're ready.

**Comment by ZeekDaGeek** [comment]
The current version is currently unplayable on servers. If you happen to cause a situation where PrimalCore spikes out and generates chunks like this:

![Image](http://i.imgur.com/SH3yRGa.png)

The chunks will keep loading forever, the world keeps getting large and larger, with no end in sight and your TPS goes down to nothing.

At the moment my server is running at 2 TPS and the world is gigs and growing.

The pack needs an emergency update.

**Comment by artdude543** [comment]
Okay. We finally have the major last issue _resolved_ so 3.1.0 is looking better for a release soon. Another Beta will be pushed beforehand, then a public release will be launched. But all the major issues are ticked off the list, thus allowing this can be closed. Any further talk can happen in Discord in regards ^_^
